pom.xml

```diff
@@ -874,9 +874,9 @@
             <version>1.2.5</version>
         </dependency>
         <dependency>
-            <groupId>com.wdtinc</groupId>
+            <groupId>io.github.ci-cmg</groupId>
```

Comment:
I took a look at this fork. If I'm not mistaken, the only thing it adds is the migration to a newer JTS version. Am I seeing this correctly? 


https://github.com/wdtinc/mapbox-vector-tile-java/compare/master...CI-CMG:mapbox-vector-tile:master

Reply:
Yes, that is correct. We use a newer version of JTS, and the previous one crashed on serializing polylines.